### PR TITLE
Add file & avatar export option to the UI

### PR DESCRIPTION
**Pull request by Daazku**
It is now possible to export files and avatar when using the vBulletin packages from the UI.
See #30

### CODE REVIEW

File: functions/render-functions.php [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
 ?>
-    <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<!DOCTYPE html>
 <html>
 <head>
     <title>Vanilla Porter - Forum Export Tool</title>
+    <meta charset="UTF-8">
     <link rel="stylesheet" type="text/css" href="style.css" media="screen"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
```
Comment (linc) [inline functions/render-functions.php]: You've just introduced the first external dependency to Porter.

**Comment by linc** [comment]
I question the need for jQuery in this project to do some simple form manipulation, but if you really want it in there, embed the whole thing in the meta rather than calling out to the Internet. Swapping around some checkboxes definitely shouldn't require an active Internet connection & that Google is unblocked (see: China).

**Comment by Daazku** [comment]
I find it way easier to do DOM manipulation with jQuery and it is usually simpler too!

**Comment by linc** [comment]
Does this still compile into 1 file with make correctly?

**Comment by Daazku** [comment]
Probably not... I totally forgot to test the UI version with that change. I will have to check for that.

**Pull request merged**